src/main/java/com/sprint/mission/discodeit/entity/UserStatus.java

```diff
@@ -0,0 +1,37 @@
+package com.sprint.mission.discodeit.entity;
+
+import lombok.Getter;
+
+import java.io.Serializable;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.UUID;
+
+@Getter
+public class UserStatus implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    private final UUID id;
+    private final Instant createdAt;
+    private Instant updatedAt;
+    private Instant lastActivityAt;
+    private final UUID userId;
+
+    public UserStatus(UUID userId) {
+        this.id = UUID.randomUUID();
+        this.userId = userId;
+        this.lastActivityAt = Instant.now();
+        this.createdAt = Instant.now();
+        this.updatedAt = Instant.now();
+    }
+
+    public void updateLastActivityAt(Instant lastActivityAt) {
```

Comment:
불필요한 인자는 제거하는게 좋을 것 같아요.  
```suggestion
    public void updateLastActivityAt() {
```